### PR TITLE
feat(svelte): add `theme` prop

### PR DESCRIPTION
**Pull request by metonym**
This PR adds a `theme` prop to the Svelte base chart to make theming easier. It also updates the documentation on using the v1 Svelte charts based on the process of updating [metonym/carbon-charts-svelte-examples](https://github.com/metonym/carbon-charts-svelte-examples/commit/8ec690a83a0778c6a60fede22d8a01a9ad4459b5).

### Updates

- feat(svelte): add `theme` prop to chart wrappers
- docs(svelte): update docs on v1 usage
- docs(svelte): update docs on theming

### Demo screenshot or recording

I've tested the updated type definitions locally.

Screen Shot 2022-06-12 at 7 06 16 AM

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/carbon-design-system/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/svelte/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ If you prefer [Yarn](https://yarnpkg.com/en/), use the following command
 instead:
 
 ```bash
-yarn add -D @carbon/charts-svelte d3
+yarn add -D @carbon/charts-svelte @carbon/styles d3
```
Comment (theiliad) [inline packages/svelte/README.md:32]: doesn't `@carbon/styles` get installed when installing `@carbon/charts-svelte`?

`@carbon/charts` I believe is a dep in `@carbon/charts-svelte`, and `@carbon/charts` depends on `@carbon/styles`

Comment (theiliad) [reply packages/svelte/README.md:32]: Also why `-D`? are these dev deps?

**Review by theiliad — CHANGES_REQUESTED** [review]
thank you @metonym. I'll have to do this for react as well, at some point 😄

File: packages/svelte/README.md (hunk 1 above)
Comment (metonym) [reply packages/svelte/README.md]: Nice catch. Resolved in https://github.com/carbon-design-system/carbon-charts/commit/aed7cf633efd7917be3092e727cd3b79d71fc414#diff-85c942869817560e818cab6cfff9460e631d0ec76ca4a24cda74b337b84f7ecf

> Also why -D? are these dev deps?

Correct. For Svelte, deps can be installed as development dependencies.

**Review by theiliad — APPROVED** [review]

**Pull request merged**